### DISCUSSION TITLE
How can i call launchPdfFromUrl in a java code?

### DISCUSSION THREAD

**Discussion by The-Pyrox** [afreakyelf/Pdf-Viewer#9] [General]
Hi, i wanted to use your library in one of our project. But i am unable to use it in .java. Could you please help?

**Comment by afreakyelf** [comment]
Hi @The-Pyrox , Thanks for checking out library.
in case of Java, to call "launchPdfFromUrl" method you need to use
PdfViewerActivity.companion.launchPdfFromUrl because library is written in kotlin.
Hope that helps.

Cheers.
